packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -47,13 +46,14 @@ class Chat extends Widget<Properties> {
   }
 
   _renderMessageList(): void {
-    const { items } = this.option();
+    const { items, user } = this.option();
 
+    const currentUserId = user?.id ?? Math.random();
```

Comment:
Why we can't add value for `id` property inside _getDefaultOptions method?

Reply:
[4d1584013e634fda124cfe9dc3d0bedad152ec3d]